File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<int, int> m;
        if(s.length()!=t.length())
            return false;
        for(int i = 0; i<s.length(); i++){
            m[s[i]-'a']++;
            m[t[i]-'a']--;
        }
        for(int i = 0; i<26; i++){
            if(m[i]!=0)
                return false;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length())
            return false;
        int[] st = new int[26];
        for (char i : s.toCharArray()) {
            st[(int) i - 'a']++;
        }
        for (char i : t.toCharArray()) {
            st[(int) i - 'a']--;
        }
        for (int i : st) {
            if (i != 0)
                return false;
        }
        return true;
    }
}